src/main/java/mods/eln/node/six/SixNodeBlock.kt

```diff
@@ -195,17 +195,17 @@ class SixNodeBlock  // public static ArrayList<Integer> repertoriedItemStackId =
         return if (sixNode.ifSideRemain) true else super.removedByPlayer(world, entityPlayer, x, y, z)
     }
 
-    override fun breakBlock(world: World, x: Int, y: Int, z: Int, par5: Block, par6: Int) {
-        if (world.isRemote == false) {
-            val tileEntity = world.getTileEntity(x, y, z) as SixNodeEntity
+    override fun breakBlock(par1World: World, x: Int, y: Int, z: Int, par5: Block, par6: Int) {
+        if (!par1World.isRemote) {
+            val tileEntity = par1World.getTileEntity(x, y, z) as SixNodeEntity
             val sixNode = tileEntity.node as SixNode? ?: return
             for (direction in Direction.values()) {
                 if (sixNode.getSideEnable(direction)) {
                     sixNode.deleteSubBlock(null, direction)
                 }
             }
         }
-        super.breakBlock(world, x, y, z, par5, par6)
+        super.breakBlock(par1World, x, y, z, par5, par6)
```

Comment:
Is it best practice to leave these with the names from the deobfuscator, or can we put this back to `world`?

Reply:
There's an issue there, because this function `override`s a base MC function then if someone calls it from the kotlin side with named arguments then it can fail and crash, the argument names are part of the public interface in kotlin and so if they don't match the overridden parent then things can crash.  Right now nothing in this code base is calling anything via named arguments, but it's something that could happen in the future and so it's just a pending bug waiting to be hit.

It can be left as 'world' if you really really want, but we'll need to quite that kotlin compiler warning and be weary of anything calling breakBlock on known instances.

Reply:
Nah, that's fine. I figured there was a technical reason why one might do that. Thanks!

Reply:
Just to be clear, it's extremely unlikely something would fail in this specific case, but it's still bad form, lol.